### PR TITLE
added the last of content

### PR DESCRIPTION
**Pull request by DannyBarker**
@MDzola @bparker12 I added the last of the content for personal. Info and internal links for his close relationships.

### CODE REVIEW

File: personal.html [hunk 1]
```diff
@@ -1,89 +1,118 @@
...
 
     <sectoin>
-        <h3>Josephine Brunsvik</h3>
-        <p></p>
+      <h3 id="karl">Karl van Beethoven</h3>
+      <h5>(1806-1858)</h5>
+      <p>
+        Ludwig had a strong father-role in Karl's life. He believed that Karl's
+        mother, Ludwig's sister, Johanna was not suited to raise Karl. The two
+        entered into an intense custody battle over Karl. Ludwig even tooka
```
Comment (MDzola) [inline personal.html]: need a space between "took" & "a"

File: personal.html [hunk 2]
```diff
@@ -1,89 +1,118 @@
...
+        break from music because he became consumed in the court. Ludwig would
+        eventually win custody, but would also come to be resented by Karl.
+        Ludwig did what he thought was best for the young boy. One of his
+        decisions was to not let the boy see his mother. Karl had ran away from
```
Comment (MDzola) [inline personal.html]: change "ran" to "run" maybe??..

File: personal.html [hunk 3]
```diff
@@ -1,89 +1,118 @@
...
     </sectoin>
 
     <section>
-        <h3>Therese Malfatti</h3>
-        <h5>(1792-1851)</h5>
-        <p></p>
+      <h3 id="therese">Therese Malfatti</h3>
+      <h5>(1792-1851)</h5>
+      <p>
+        Beethoven was enfatuated with Therese Malfatti for a period of time. It
```
Comment (MDzola) [inline personal.html]: change "enfatuated" to "infatuated"

**Review by MDzola — CHANGES_REQUESTED** [review]
just a few spelling changes. Looks awesome otherwise!

**Review by MDzola — APPROVED** [review]
Looks Great! 👍

File: personal.html [hunk 4]
```diff
@@ -1,89 +1,118 @@
...
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Document</title>
```
Comment (bparker12) [inline personal.html]: Add Title

**Review by bparker12 — CHANGES_REQUESTED** [review]

**Review by bparker12 — APPROVED** [review]
👍

**Pull request merged**